src/lib/backend.ai-client-node.ts

```diff
@@ -1060,7 +1062,7 @@ class Client {
 
   // legacy aliases (DO NOT USE for new codes)
   createKernel(kernelType, sessionId = undefined, resources = {}, timeout = 0) {
-    return this.createIfNotExists(kernelType, sessionId, resources, timeout);
+    return this.createIfNotExists(kernelType, sessionId, 'x86_64', resources, timeout);
```

Comment:
I'm not sure whether it's a good idea to use hard-coded `x86_64` value here.
How about adding key-value in config file instead?

Reply:
Also, the parameter order should be kept for compatibility. @kyujin-cho

Reply:
Fixed.

Reply:
@lizable It is for legacy code. So only x86_64 will come into the role.